### PR TITLE
Storage concurrency

### PR DESCRIPTION
**Pull request by jforissier**

### CODE REVIEW

File: core/tee/tee_ree_fs.c [hunk 1]
```diff
@@ -138,6 +138,8 @@ struct block_operations {
 
 static struct handle_db fs_handle_db = HANDLE_DB_INITIALIZER;
 
+static struct mutex mutex = MUTEX_INITIALIZER;
```
Comment (jenswi-linaro) [inline core/tee/tee_ree_fs.c]: Even though the three different static variables `mutex` have nothing to do with each other and are static, it's still a bit confusing.

How about `ree_fs_mu` or `ree_fs_mutex` instead in this file for instance?

Comment (jforissier) [inline core/tee/tee_ree_fs.c]: Yep that would be more reasonable indeed. Will fix.

File: core/tee/tee_sql_fs.c [hunk 2]
```diff
@@ -83,6 +84,8 @@ struct tee_fs_dir {
 /* Container for file descriptors (struct sql_fs_fd) */
 static struct handle_db fs_db = HANDLE_DB_INITIALIZER;
 
+static struct mutex mutex = MUTEX_INITIALIZER;
```
Comment (jenswi-linaro) [inline core/tee/tee_sql_fs.c]: Please rename the variable to `sql_fs_mutex` or similar.

Comment (jforissier) [inline core/tee/tee_sql_fs.c]: Yes, sure

**Comment by jenswi-linaro** [comment]
`Reviewed-by: Jens Wiklander <jens.wiklander@linaro.org>`

**Pull request merged**